ui/src/components/SelectData.ts

```diff
@@ -0,0 +1,113 @@
+export interface KeyOption {

```

Comment:
i dunno what this file is but it looks copied off of a canadian who spells flavor wrong lol.

Anyways, just a reminder to keep tasks in separate PRs instead of grouping them. The key stuff should be separate from the dropzone stuff because they are 2 separate things. If a tiny piece of one doesn't work, it ends up blocking 2 tasks instead of one. Or, if in the future we find a bug in one thing, we cant just pull it out of production, we would have to pull 2 things out unnecessarily. So if you can, put the select stuff in a separate PR than the dropzone stuff